src/coursescheduler/temp_json_input/schedule_object.json

```diff
@@ -14,11 +14,13 @@
                   "sections": [
                         {
                               "professor": null,
+                              "maxCapacity": null,

```

Comment:
maxCapacity is not in the alg specs. It is going to be implemented by Alg2 team right?

Reply:
Yes, it is not in the alg spec because it is not aligned with company 1. Our alg 2 will use it if it's there but will support the current spec (without `maxCapacity`) for company 1's use.